shared/actions/pinentry.js

```diff
@@ -28,8 +27,28 @@ export function registerPinentryListener (): AsyncAction {
       payload: {started: true},
     }: RegisterPinentryListenerAction))
 
-    const pinentryListeners = pinentryListenersCreator(dispatch)
-    engine.listenGeneralIncomingRpc(pinentryListeners)
+    engine.setIncomingHandler('keybase.1.secretUi.getPassphrase', (payload, response) => {
```

Comment:
:sheep: Might be valuable to give this function a name (rather than arrow function) for stack traces